pkgs/applications/video/vlc/default.nix

```
@@ -52,6 +54,7 @@ stdenv.mkDerivation rec {
     fluidsynth wayland wayland-protocols
   ] ++ optional (!stdenv.hostPlatform.isAarch64) live555
     ++ optionals withQt5    [ qtbase qtsvg qtx11extras ]
+    ++ optionals skins2Support [ libXpm freetype libXext libXinerama ]
```

```suggestion
    ++ optionals skins2Support with xorg; [ libXpm freetype libXext libXinerama ]
```

This way, you don't need to add `libXpm`, `libXext` and `libXinerama` as parameters (only `freetype`).